.github/ISSUE_TEMPLATE/1-Bug_report.md

```diff
@@ -15,7 +15,7 @@ labels: 'bug'
 - [ ] Using yarn
```

Comment:
While we're updating this, we should also ask the issue creator to donate to the project and point them to https://github.com/electron-react-boilerplate/electron-react-boilerplate/issues/2308